src/System.Management.Automation/engine/CmdletParameterBinderController.cs

```diff
@@ -1684,33 +1684,31 @@ private void HandleRemainingArguments()
                         var cpi = CommandParameterInternal.CreateParameterWithArgument(
                             PositionUtilities.EmptyExtent, varargsParameter.Parameter.Name, "-" + varargsParameter.Parameter.Name + ":",
                             argumentExtent, valueFromRemainingArguments, false);
+
+                        // To make all of the following work similarly (the first is handled elsewhere, but second and third are
+                        // handled here):
+                        //     Set-ClusterOwnerNode -Owners foo,bar
+                        //     Set-ClusterOwnerNode foo bar
+                        //     Set-ClusterOwnerNode foo,bar
+                        // we unwrap our List, but only if there is a single argument of type object[].
+                        if (valueFromRemainingArguments.Count == 1 && valueFromRemainingArguments[0] is object[])
```

Comment:
Should it be `valueFromRemainingArguments[0]?.GetType() == typeof(object[])` or `valueFromRemainingArguments[0] is Array`?
Given array covariance in .NET, any array of reference type (like `string[]`) will pass `is object[]` check, but not array of value type (like `int[]`). Should it be only `object[]` or all arrays instead?

Reply:
@PetSerAl - good question, I think the intent was all arrays, though `object[]` in this context is nearly the same thing.

@dlwyatt , @daxian-dbw - thoughts?

Reply:
Looks like you're right.  Most of the time, PowerShell's building `object[]` arrays from arguments at that point, but if you explicitly make a variable of another type and pass it in, it doesn't get unwrapped.

Rather than `is Array`, though, I'd recommend using `LanguagePrimitives.GetEnumerable(valueFromRemainingArguments[0])` .  That will cover other types of collections using the same rules that you'd expect from PowerShell (not treating strings / dictionaries as collections, etc.)

Reply:
I'm putting together new tests for this and will follow up with a second PR